### PR TITLE
Fixed the english of a system message

### PR DESCRIPTION
**Pull request by MrAuro**
# Description

Changed a system message from "sending messages too fast" to "You are sending messages too fast". :)

### CODE REVIEW

**Comment by Mm2PL** [comment]
Your PR makes the rate limit messages different from each other. Can you change the second one too? 
```cpp
auto errorMessage =

makeSystemMessage("sending too many messages");
```
I think you should also add a changelog entry for this change :)

File: src/providers/twitch/TwitchIrcServer.cpp [hunk 1]
```diff
@@ -335,8 +335,9 @@ void TwitchIrcServer::onMessageSendRequested(TwitchChannel *channel,
         {
             if (this->lastErrorTimeSpeed_ + 30s < now)
             {
+                // this was "sending messages too fast"
```
Comment (leon-richardt) [inline src/providers/twitch/TwitchIrcServer.cpp:338]: This comment can be removed.

**Review by leon-richardt — CHANGES_REQUESTED** [review]

**Comment by RAnders00** [comment]
it's still not the right english expression, if anything it should be "you are sending messages too quickly" (need an adverb, not adjective)

File: src/providers/twitch/TwitchIrcServer.cpp [hunk 2]
```diff
@@ -336,7 +336,7 @@ void TwitchIrcServer::onMessageSendRequested(TwitchChannel *channel,
             if (this->lastErrorTimeSpeed_ + 30s < now)
             {
                 auto errorMessage =
-                    makeSystemMessage("sending messages too fast");
+                    makeSystemMessage("You are sending messages too fast.");
```
Comment (RAnders00) [inline src/providers/twitch/TwitchIrcServer.cpp:339]:
```suggestion
                    makeSystemMessage("sending messages too quickly");
```

File: src/providers/twitch/TwitchIrcServer.cpp [hunk 3]
```diff
@@ -357,7 +357,7 @@ void TwitchIrcServer::onMessageSendRequested(TwitchChannel *channel,
             if (this->lastErrorTimeAmount_ + 30s < now)
             {
                 auto errorMessage =
-                    makeSystemMessage("sending too many messages");
+                    makeSystemMessage("You are sending too many messages.");
```
Comment (RAnders00) [inline src/providers/twitch/TwitchIrcServer.cpp:360]:
```suggestion
                    makeSystemMessage("sending too many messages");
```

File: src/providers/twitch/TwitchIrcServer.cpp (hunk 2 above)
Comment (MrAuro) [reply src/providers/twitch/TwitchIrcServer.cpp:339]: My change was to make it into a sentence from a sentence fragment. However from "too fast" to "too quickly" sounds better.

**Comment by RAnders00** [comment]
the changelog entry has been merged into the already released section for v2.2.0, not into the unreleased section 👀 

image

**Review by RainbowScientist5 — COMMENTED** [review]
image image](https://github.com/Chatterino/chatterino2/assets/130253379/4c60b76a-03a6-4f3b-943b-1f9931e3d078)/assets/130253379/005ddf2a-37a1-42a2-8697-ea66b432c46c)

**Pull request merged**